library/std/src/prelude/v1.rs

```diff
@@ -54,33 +54,30 @@ pub use core::prelude::v1::{
 #[doc(no_inline)]
 pub use core::prelude::v1::concat_bytes;
 
-// FIXME: Attribute and internal derive macros are not documented because for them rustdoc generates
-// dead links which fail link checker testing.
+// Do not `doc(inline)` these `doc(hidden)` items.
 #[stable(feature = "builtin_macro_prelude", since = "1.38.0")]
-#[allow(deprecated, deprecated_in_future)]
-#[doc(hidden)]
-pub use core::prelude::v1::{
-    bench, global_allocator, test, test_case, RustcDecodable, RustcEncodable,
-};
+#[allow(deprecated)]
+pub use core::prelude::v1::{RustcDecodable, RustcEncodable};
 
+// Do not `doc(no_inline)` so that they become doc items on their own
+// (no public module for them to be re-exported from).
```

Comment:
FWIW, for std there is such a public module, it's `core::prelude::v1`.
So maybe in std macros should keep this `doc(no_inline)`, like any other non-macro items.

Reply:
So, I was having this weird error: https://github.com/rust-lang/rust/pull/92456#issuecomment-1003435516. I can try `no_inline`-ing again, but I suspect it will do the same? I'll admit I don't understand why it fails: could that be some `bootstrap`-related error?

Reply:
>could that be some bootstrap-related error?

Very unlikely.
Looks like rustdoc still points to the original definition module even if it was "inlined away".
Let's keep this as is then.